mobile/bindings.go

```diff
@@ -53,9 +70,15 @@ func Start(extraArgs string, callback Callback) {
 		}
 	}()
 
-	// TODO(halseth): callback when RPC server is actually running. Since
-	// the RPC server might take a while to start up, the client might
-	// assume it is ready to accept calls when this callback is sent, while
-	// it's not.
-	callback.OnResponse([]byte("started"))
+	// Finally we start two go routines that will call the provided
+	// callbacks when the RPC servers are ready to accept calls.
+	go func() {
```

Comment:
Maybe a simpler approach could be to modify `lnd.Main()` by adding a `started` channel as an incoming argument which then could be used to signal that all initialization is finished and we're waiting on	`<-signal.ShutdownChannel()`. WDYT?

Reply:
Because the unlocker is conditional (it's only started if the wallet is actually locked) you need to know what the state of the individual servers is to be able to react. So there would need to be two `started` channels which basically comes down to the implemented solution.
I think this is probably the most simple you can get with the current architecture.

Reply:
I see, thanks for the clarificaton @guggero .

Reply:
👍